examples/custom-widget-with-3rd-party-data/src/index.ts

```diff
@@ -0,0 +1,24 @@
+import { client } from './client';
+import { renderController } from './controller';
+import { renderCustomWidget } from './custom-widget';
+import { renderLogin } from './login';
+
+switch (window.location.pathname) {
+    case '/custom-widget':
+        renderCustomWidget(client);
+        break;
+
+    /**
+     * This route is not really part of the App.
+     * It's only provided for completeness.
+     * In an actual App,
+     * this should be removed and the actual authentication route should be used.
+     */
```

Comment:
Login *could* be part of the app if they wanted it to be. Maybe the right wording is about that option? So something like 'Here the login route is part of the datadog app but the system is designed to be able to leverage  preexisting login route if you have access to it'